settings.gradle.kts

```diff
@@ -38,6 +38,7 @@ include(":subprojects:gradle:graphite-config")
 include(":subprojects:gradle:impact")
 include(":subprojects:gradle:impact-shared")
 include(":subprojects:gradle:module-dependencies-graph")
+include(":subprojects:gradle:module-dependencies-shared")
```

Comment:
I'm not confident in that name. It would be good If you could replace `shared` with something meaningful

Reply:
Agree, I'll try to find more obvious name

Reply:
#1298